models/AdminSchema.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, model } from 'mongoose';
+
+const AdminSchema = new Schema(
+  {
+    email: {
+      type: String,
+      required: true,
+      max: 50,
```

Comment:
Hej, e-mail zwiekszyłabym do 255, poza tym wyglada super :)

Reply:
tak jak w feature/hr-schema, zostawił bym to tak lub zwiększył max 64, do uzgodnienia